detect_secrets/core/secrets_collection.py

```diff
@@ -176,7 +188,7 @@ def __iter__(self) -> Generator[Tuple[str, PotentialSecret], None, None]:
         for filename in sorted(self.files):
             secrets = self[filename]
 
-            # TODO: Handle cases when line numbers are not supplied
+            # NOTE: If line numbers aren't supplied, they will default to 0.
```

Comment:
would prefer `None` to be the line number value if not provided but not a big deal